packages/forklift-console-plugin/src/modules/Plans/views/list/components/ActionsCell.tsx

```diff
@@ -23,46 +15,20 @@ export const ActionsCell = ({ data }: CellProps) => {
   const { showModal } = useModal();
 
   const plan = data.obj;
-
-  const canStart = canPlanStart(plan);
-  const canReStart = canPlanReStart(plan);
-
   const isWarmAndExecuting = plan?.spec?.warm && isPlanExecuting(plan);
   const isArchived = isPlanArchived(plan);
 
-  const buttonStartLabel = canReStart ? t('Restart') : t('Start');
-  const buttonStartIcon = canReStart ? <ReStartIcon /> : <StartIcon />;
-  const buttonCutoverIcon = <CutoverIcon />;
-
-  const onClickPlanStartMigration = () => {
-    showModal(
-      <PlanStartMigrationModal resource={data.obj} model={PlanModel} title={buttonStartLabel} />,
-    );
-  };
-
   const onClickPlanCutoverMigration = () => {
-    showModal(<PlanCutoverMigrationModal resource={data.obj} />);
+    showModal(<PlanCutoverMigrationModal resource={plan} />);
   };
 
   return (
     <Flex flex={{ default: 'flex_3' }} flexWrap={{ default: 'nowrap' }}>
       <FlexItem grow={{ default: 'grow' }}></FlexItem>
 
-      {canStart && (
-        <FlexItem align={{ default: 'alignRight' }}>
-          <Button variant="secondary" icon={buttonStartIcon} onClick={onClickPlanStartMigration}>
-            {buttonStartLabel}
-          </Button>
-        </FlexItem>
-      )}
-
       {isWarmAndExecuting && !isArchived && (
         <FlexItem align={{ default: 'alignRight' }}>
-          <Button
-            variant="secondary"
-            icon={buttonCutoverIcon}
-            onClick={onClickPlanCutoverMigration}
-          >
+          <Button variant="secondary" icon={<CutoverIcon />} onClick={onClickPlanCutoverMigration}>
```

Comment:
I guess we will remove this in a followup ...

Reply:
Right, at least for now, I think the design plan is to move this button to a "Migration type" column as seen here: 
<img width="540" alt="image" src="https://github.com/user-attachments/assets/6509ba86-5300-491b-b32c-cb6bcae1d423" />
(can be seen in figma [here](https://www.figma.com/proto/vmP21XkRQojF17AEKPUA83/MTV-Monitoring-v1?page-id=0%3A1&node-id=410-13931&node-type=canvas&viewport=-2582%2C-1060%2C1&t=mN2Fteu99bXwKyT0-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=410%3A13931&show-proto-sidebar=1))